src/app.html

```diff
@@ -13,12 +13,10 @@
 		/>
 		%sveltekit.head%
 	</head>
-	<body
-		class="gruv:text-gray-100 text-gray-100 bg-gradient-to-r gruv:from-orange-800 gruv:to-yellow-800 from-neutral-900 to-slate-900"
```

Comment:
Maybe removing `from-neutral-900 to-slate-900` here created this issue https://github.com/codicocodes/dotfyle/pull/74#issuecomment-1793780927